database/migrations/2016_11_18_200000_create_combined_fragments_table.php

```diff
@@ -9,11 +10,10 @@ public function up()
     {
         Schema::create('combined_fragments', function (Blueprint $table) {
             $table->increments('id');
-            $table->timestamps();
+            $table->timestamp('timestamps');
```

Comment:
nie to som ti skopíroval z dokumentácie lebo si tam mal ` $table->timestamp('timestamps');`
to čo si napísal vytvorí stĺpec s názvom 'timestamps' databázového typu timestamp, ale my chceme
`$table->timestamps();` čo vytvorí dva stĺpce, jeden s menom 'created_at' a druhý 'updated_at', oba typu timestamp a s povolenou nulovou hodnotou